packages/wonder-blocks-breadcrumb/components/breadcrumb.md

```diff
@@ -0,0 +1,10 @@
+### breadcrumb example (with links):
```

Comment:
Would be nice to show one here that's a mix of links and plaintext, too.

Reply:
I think this example can be moved into breadcrumb/docs.md and you can get rid of these per-component examples as people will mostly just be interested in how to use this at a full component level.